flexget/plugins/cli/series_list.py

```diff
@@ -0,0 +1,344 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import re
+from argparse import ArgumentParser, ArgumentTypeError
+
+from sqlalchemy.orm.exc import NoResultFound
+
+from flexget import options, plugin
+from flexget.event import event
+from flexget.logger import console
+from flexget.manager import Session
+from flexget.utils import qualities
+from flexget.config_schema import parse_interval
+from flexget.plugins.list.series_list import SeriesListList, SeriesListDBContainer as slDb, SeriesList, get_db_series
+from flexget.plugins.filter.series import FilterSeriesBase
+
+SETTINGS_SCHEMA = FilterSeriesBase().settings_schema
+SERIES_ATTRIBUTES = SETTINGS_SCHEMA['properties']
+
+
+def supported_ids():
```

Comment:
you don't need this duplication, you can access this via FilterSeriesBase.supported_ids .. correct?